README.md

```diff
@@ -1 +1,26 @@
-# hello-world-4
\ No newline at end of file
+# hello-world-4
+
+> Description of the project.
+
+
```

Comment:
- [ ] Kindly Use the Microverse [template ](https://github.com/microverseinc/readme-template)for your Readme file, and Microverse Logo before the title and the author's social media links are missing.